packages/apputils/style/commandpalette.css

```diff
@@ -150,7 +150,7 @@
 }
 
 .p-CommandPalette-item.p-mod-disabled mark {
-  color: rgba(0, 0, 0, 0.4);
```

Comment:
The font color here should be "--jp-ui-font-color3" to work in both the light and dark themes

Reply:
Thx I will check it out and upload a new commit